app/data_source/api/basic_document.py

```diff
@@ -45,7 +49,7 @@ class BasicDocument:
     data_source_id: int  # data source id in database
     type: DocumentType
     title: str
-    content: str
+    content: Union[str, List[PDFDocument]]
```

Comment:
should be str.

Reply:
@Roey7 reverted and changed back to as it was `str`